zenfs.mk

```diff
@@ -1,6 +1,7 @@
 zenfs_SOURCES = fs/fs_zenfs.cc fs/zbd_zenfs.cc fs/io_zenfs.cc
 zenfs_HEADERS = fs/fs_zenfs.h fs/zbd_zenfs.h fs/io_zenfs.h fs/version.h fs/metrics.h fs/snapshot.h
 zenfs_LDFLAGS = -u zenfs_filesystem_reg
+zenfs_TEST_MAIN_SOURCES = fs/fs_zenfs_test.cc
```

Comment:
I think we should introduce a test/ directory with the tests in stead

Reply:
Moved tests to subdir